app/src/main/java/org/supla/android/lib/SuplaClient.java

```diff
@@ -73,13 +74,15 @@ public class SuplaClient extends Thread {
     private String oneTimePassword;
     private long _connectingStatusLastTime;
     private final ProfileManager profileManager;
+    private final WidgetVisibilityHandler widgetVisibilityHandler;
 
-    public SuplaClient(Context context, String oneTimePassword, ProfileManager profileManager) {
+    public SuplaClient(Context context, String oneTimePassword, ProfileManager profileManager, WidgetVisibilityHandler widgetVisibilityHandler) {
```

Comment:
A suggestion to manually break long lines (here and in other places) and try to have them less than 80 characters long. Otherwise the readability will suffer if viewed in any other tool than AndroidStudio/Intellij

Reply:
I've changed my configuration to 80, but it's quite short